substrate-node/.maintain/benchmark.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
I see this script is a helper to facilitate the`weights.rs` files generation.
Great! The missing benchmarks will come soon
Is it called somewhere or is it to be manually executed?

Reply:
It could be automated, but for now it's called manually.